src/platforms/dart/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -21,3 +21,13 @@ The span finishes once the request has been executed. The span `status` depends
 When the HTTP request throws an `Exception`, Sentry's SDK associates this exception to the running span. If you haven't set the SDK to swallow the exception and capture it, the span and SentryEvent will be linked when viewing it on the **Issue Details** page in sentry.io.
 
 For more information see our [SentryHttpClient integration](/platforms/dart/configuration/integrations/http-integration/#performance-monitoring-for-http-requests).
+
+### Dio Library Instrumentation
```

Comment:
Perhaps easier for those who don't know about Dio, to know this is an HTTP library

```suggestion
### Dio HTTP Library Instrumentation
```